.travis.yml

```diff
@@ -5,20 +5,35 @@ rvm:
   - 2.3
   - 2.4
   - 2.5
+  - 2.6.2
```

Comment:
Do we want a specific 2.6 version here, or just 2.6?

Reply:
Yes, I just want the latest 2.6. I don't remember why I was this specific.
I will try using 2.6 and see if that will do the job.

FYI The travis ruby list is here is anyone wants to see what I am working from
http://rubies.travis-ci.org/

Reply:
2.6 works just fine. I don't know why I was more specific. 🤷‍♂